### ISSUE TITLE
can you make it be a book?

### ISSUE THREAD

**Issue by aexftf** [donnemartin/system-design-primer#691]

**Comment by rcalixte** [comment]
@aexftf There is an open PR for adding automation to generate a PDF after a new commit is merged.

Until then, can you check out the PDF [here](https://github.com/donnemartin/system-design-primer/issues/4#issuecomment-1001191308) and share any feedback? It should be an exact duplicate of what will be generated once the PR is merged.

**Comment by 789chandan** [comment]
yes, can make it. We can convert it in pdf.
